bluetooth: Bluetooth GATT server serialization (#4984)
nrfconnect/sdk-nrf: include/bluetooth/rpc.rst

The full Bluetooth LE stack can run on another device or CPU, such as the nRF5340 DK network core using :ref:`nrfxlib:nrf_rpc`.

.. note::
   The |NCS| currently supports serialization of the :ref:`zephyr:bt_gap`, the :ref:`zephyr:bluetooth_connection_mgmt` and the GATT server only.

>    Due to the limited support, when :option:`CONFIG_BT_GATT_CLIENT` option is disabled then it allows enabling the :option:`CONFIG_BT_RPC` option.

Reviewer: ```suggestion
   Due to limited support, enabling of :option:`CONFIG_BT_RPC` option is allowed first when :option:`CONFIG_BT_GATT_CLIENT` option is disabled.
```
